myokit/formats/sbml/_importer.py

```diff
@@ -159,42 +188,101 @@ def model(self, path):
         # Return finished model
         return model
 
-    def _parse_initial_assignments(self, model, comp, refs, node):
+    def _get_namespaces(self, sbml_version: str, log):
+        """
+        Defines namespaces for supported SBML versions. Supported versions are:
+        level 2, version 3, 4, 5.
+        """
+        ns = dict()
+        if sbml_version == "{http://www.sbml.org/sbml/level2/version3}":
+            # SBML
+            ns['sbml'] = "{http://www.sbml.org/sbml/level2/version3}"
+            # MathML
+            ns['mathml'] = "{http://www.w3.org/1998/Math/MathML}"
```

Comment:
Does this change in past or future versions? Otherwise can just set once for all versions, I think

Reply:
This won't change, but other properties such as base unit definitions, and other namespaces. For now we could, however, just add the 'mathml' namespace irrespective of the sbml version

Reply:
Done.